jib-core/src/integration-test/java/com/google/cloud/tools/jib/api/ContainerizerIntegrationTest.java

```diff
@@ -17,11 +17,11 @@
 package com.google.cloud.tools.jib.api;
 
 import com.google.cloud.tools.jib.Command;
-import com.google.cloud.tools.jib.builder.BuildStepType;
```

Comment:
I don't think we need to expose `BuildStepType` either.

Reply:
`BuildStepType` is used in `LayerCountEvent`.

`.event.events` is mainly where I put the event types themselves, and `.event` was for everything else. I can combine them if you want, but I kinda liked having all the event types isolated in their own package (especially if we get rid of `JibEventType`). But since there aren't that many maybe we don't need separate packages.

Reply:
`.api` had been flat until this.  If we want to go this direction then might make more sense to invert and use the `.internal`.

Reply:
Hmm that's true. I'll make it completely flat for now until we figure out what we want in the public api/fix all the unintended exposed stuff, then maybe we can do another pass where we organize into sub-packages/use `.internal` for everything else.